Jenkinsfile

```diff
@@ -21,6 +21,21 @@ pipeline {
       }
     }
 
+    stage('clear deployment after subsystem test') {
+      steps {
+        sh 'make clear-deployment'
+      }
+    }
+
+    stage('test subsystem with podman') {
```

Comment:
I think that `clean-onprem` should be called
Maybe even as a first stage
Otherwise the next job may experience some strange things.

Reply:
Nice catch. I've added clean-onprem to the first stage.